src/puppetlabs/puppetdb/queue.clj

```diff
@@ -250,21 +250,32 @@
                  file-extension))))
 
 (defn cmdref->cmd [q cmdref]
+  "Returns the command associated with cmdref, or logically false if
+  the file is missing (i.e. someone has deleted it)."
```

Comment:
For what it's worth, by default I tend to leave that unspecified in documentation so that the result is free to change later if that better suits a future implementation, and it also reminds not to use `(nil? result)` -- though arguably such code changes may be more likely in languages with additional false values (i.e. `return x[i] - y[i]`, etc.).